.cursor/rules/DataCollection, Submission, Binding.md

```
@@ -0,0 +1,393 @@
+
+
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)